Makefile

```diff
@@ -112,9 +115,13 @@ $(BUILD)/test.out: $(BUILD)/test.js | $(BUILD)
 	mv $@.tmp $@ # Move the output where it belongs.
 
 $(FORMAT)/%.purs.formatted: %.purs $(NODE_MODULES)
+ifeq '$(USER)' 'root'
+	@echo "$(RED)purty should not be ran as root, skipping"
```

Comment:
"should not be _run_ as root", but otherwise 👍

Reply:
🙃